Harmonize PET tracers handling (ML / DL / Stats)

alexandreroutier opened the pull request:
Follows PR #129.

After this PR, handling of PET tracers will be the same across Clinica pipelines (`acq_label` / `suvr_reference_region` / optional `use_pvc_data`).

alexandreroutier commented:
Hi @mdiazmel,

This PR is now ready for technical review. :)

I will finish ASAP the documentation so that @nburgos can do the Wiki review.

Alex

nburgos reviewed (commented):
I checked all the wiki pages and made modifications when needed. Should be all good now.

mdiazmel commented on clinica/utils/inputs.py:
```diff
@@ -447,13 +447,29 @@ def fetch_file(remote, dirname=None):
+    Returns:
+        Path to downloaded file.
+
+    Note:
```
To remove!

mdiazmel commented on clinica/utils/inputs.py:
```diff
@@ -447,13 +447,29 @@ def fetch_file(remote, dirname=None):
+    else:
+        cache_clinica = os.path.join(home, ".cache", "clinica", "data")
     if not (os.path.exists(cache_clinica)):
         os.makedirs(cache_clinica)
```
Add the trick of `exist_ok=True` to avoid the `if` instruction.

mdiazmel reviewed (commented):
Thanks @alexandreroutier,
Code readability has improved a lot with these changes!
A couple suggestions before merging.

alexandreroutier commented:
> Thanks @alexandreroutier,
> Code readability has improved a lot with these changes!
> A couple suggestions before merging.

Hi!

Thanks for the feedback, I addressed the comments in 022b72e :)

I think this can now be merged.

mdiazmel commented:
Hi @alexandreroutier ,
I propose to do a "squash and merge" for this PR, if you're agree.
